tests/simulationbase.py

```diff
@@ -211,23 +208,15 @@ def _check_results(
 
     @pytest.mark.asyncio
     async def test_provider_shutdown_in_benchmark(self, task_lifecycle_util):
-        benchmark_defer = asyncio.ensure_future(self._simulate_cube_task(
+        benchmark_future = asyncio.ensure_future(self._simulate_cube_task(
             1,
             self._get_cube_params("1", resolution=[10000, 6000]),
             task_lifecycle_util,
             [1],
         ))
 
-        async def _shutdown_in_5s():
-            await asyncio.sleep(5.0)
-            await task_lifecycle_util.shutdown_provider()
-            return None
-        shutdown_defer = asyncio.ensure_future(_shutdown_in_5s())
-
-        done, _ = await asyncio.wait(
```

Comment:
The goal here was to also start the benchmark before calling shutdown, will this still be the case?

Reply:
Yes